### PR TITLE
feat: Seek behavior

### PR DESCRIPTION
**Pull request by st-pasha**
This is a somewhat preliminary implementation of the `Seek` behavior for two different types of `Kinematics`.

Examples/tests will be added later, when I figure out a system for them.

### CODE REVIEW

File: lib/src/steering/kinematics/max_acceleration_kinematics.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import 'package:vector_math/vector_math_64.dart';
+
+import '../behaviors/seek.dart';
+import '../kinematics.dart';
+
+class MaxAccelerationKinematics extends Kinematics {
```
Comment (spydon) [inline lib/src/steering/kinematics/max_acceleration_kinematics.dart:15]: Missing dartdocs

File: lib/src/steering/kinematics/max_speed_kinematics.dart [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import 'package:vector_math/vector_math_64.dart';
+
+import '../behaviors/seek.dart';
+import 'basic_kinematics.dart';
+
+/// [MaxSpeedKinematics] describes a character who can move freely in any
+/// direction with speeds up to `maxSpeed`, and change their velocity
+/// instantaneously with no regard to inertia.
+///
```
Comment (spydon) [inline lib/src/steering/kinematics/max_speed_kinematics.dart:9]: Extra line

**Review by spydon — APPROVED** [review]
Lgtm, maybe pull in Melos so that we don't have to manage the changelog manually?

**Comment by st-pasha** [comment]
> Lgtm, maybe pull in Melos so that we don't have to manage the changelog manually?

I was thinking of adding Melos after the initial release, because the initial changelog wouldn't be very meaningful anyways.

**Pull request merged**